automation_tools/__init__.py

```diff
@@ -445,9 +445,19 @@ def install_puppet_scap_client():
     run('yum -y install puppet-foreman_scap_client', warn_only=True)
 
 
-def install_discovery_image():
-    """Task to install foreman discovery image."""
+def setup_foreman_discovery():
+    """Task to setup foreman discovery."""
+    admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+    template_url = os.environ['PXE_DEFAULT_TEMPLATE_URL']
+    template_file = run('mktemp /var/tmp/template.XXXXXXXXXXXX')
```

Comment:
Is there any reason to specifically create a temp file in `/var/tmp` instead of, say, `/tmp`? If there's no special reason to choose one directory over the other, I'd suggest using just `mktemp`, like so:

```
[root@qe-sat6-rhel66 ~]# mktemp
/tmp/tmp.t9hWUeagql
```